doc/source/data/batch_inference.rst

```diff
@@ -467,7 +467,7 @@ Models that have been trained with :ref:`Ray Train <train-docs>` can then be use
 
 **Step 3:** Use Ray Data for batch inference. To load in the model from the :class:`Checkpoint <ray.train.Checkpoint>` inside the Python class, use one of the framework-specific Checkpoint classes.
```

Comment:
Need to change this paragraph as well. No longer using framework specific Checkpoint classes.

Let’s link to the appropriate sections for loading models for TorchTrainer and other trainers?

Reply:
Ah good catch, will update this.